packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.Animation.Example.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import { DetailsList, DetailsListLayoutMode, IColumn } from 'office-ui-fabric-react/lib/DetailsList';
+import { Fabric } from 'office-ui-fabric-react/lib/Fabric';
+
+export interface IDetailsListAnimationExampleItem {
+  key: number;
+  name: string;
+  value: number;
+}
+
+export interface IDetailsListAnimationExampleState {
+  items: IDetailsListAnimationExampleItem[];
+}
+
+export class DetailsListAnimationExample extends React.Component<{}, IDetailsListAnimationExampleState> {
+  private _allItems: IDetailsListAnimationExampleItem[];
+  private _columns: IColumn[];
+
+  constructor(props: {}) {
+    super(props);
+
+    // Populate with items for demos.
+    this._allItems = [];
+    for (let i = 0; i < 200; i++) {
+      this._allItems.push({
+        key: i,
+        name: 'Item ' + i,
+        value: i
+      });
+    }
+
+    const LOREM_IPSUM = (
+      'lorem ipsum dolor sit amet consectetur adipiscing elit sed do eiusmod tempor incididunt ut ' +
+      'labore et dolore magna aliqua ut enim ad minim veniam quis nostrud exercitation ullamco laboris nisi ut ' +
+      'aliquip ex ea commodo consequat duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore ' +
+      'eu fugiat nulla pariatur excepteur sint occaecat cupidatat non proident sunt in culpa qui officia deserunt '
+    ).split(' ');
+    let loremIndex = 0;
+    function _lorem(wordCount: number): string {
+      const startIndex = loremIndex + wordCount > LOREM_IPSUM.length ? 0 : loremIndex;
+      loremIndex = startIndex + wordCount;
+      return LOREM_IPSUM.slice(startIndex, loremIndex).join(' ');
+    }
+    setInterval(() => {
```

Comment:
This timer needs to be cleared on `componentWillUnmount`

Reply:
You might also want to defer setting it until `componentDidMount` instead of the constructor